### PR TITLE
Update translation files for 3.8.0rc1

### PR DESCRIPTION
**Pull request by softins**
Includes a few minor updates from the source (and a lot of line number changes).

Also tagged as "unfinished" all translations of "Central Server" that
did not appear obviously to have been updated to "Directory Server".

These will need a round-trip to the translators before 3.8.0

### CODE REVIEW

**Review by hoffie — APPROVED** [review]
Thanks for doing this -- sounds tedious, but should save a whole bunch of time and work elsewhere!

All of it looks good to me:
- All German changes sound correct.
- Changes in all other languages look consistent as well
- "unfinished" status looks good across all languages, too (newly added What's this texts are expected to be empty and unfinished, manually translated/updates trings are expected to stay unfinished)

File: src/res/translation/translation_fr_FR.ts [hunk 1]
```diff
@@ -2293,64 +2319,78 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
...
-        <location filename="../../util.cpp" line="586"/>
+        <location filename="../../util.cpp" line="597"/>
         <source>What&apos;s &amp;This</source>
         <translation>Qu&apos;est-ce que c&apos;es&amp;t</translation>
     </message>
     <message>
-        <location filename="../../util.cpp" line="588"/>
+        <location filename="../../util.cpp" line="599"/>
+        <source>&amp;About Jamulus...</source>
+        <translation type="unfinished">À &amp;propos Jamulus...</translation>
```
Comment (jujudusud) [inline src/res/translation/translation_fr_FR.ts:2345]: Should be: "À &amp;propos de Jamulus..."

File: src/res/translation/translation_fr_FR.ts [hunk 2]
```diff
@@ -2293,64 +2319,78 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
...
+    </message>
+    <message>
+        <location filename="../../util.cpp" line="601"/>
+        <source>About &amp;Qt...</source>
+        <translation type="unfinished">À propos &amp;Qt...</translation>
```
Comment (jujudusud) [inline src/res/translation/translation_fr_FR.ts:2350]: should be: "À propos de &amp;Qt..."

File: src/res/translation/translation_fr_FR.ts [hunk 3]
```diff
@@ -2293,64 +2319,78 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
...
+    </message>
+    <message>
         <source>&amp;About...</source>
-        <translation>&amp;À propos...</translation>
+        <translation type="vanished">&amp;À propos...</translation>
+    </message>
+    <message>
+        <location filename="../../util.h" line="441"/>
+        <source>About Qt</source>
+        <translation type="unfinished">À propos Qt</translation>
```
Comment (jujudusud) [inline src/res/translation/translation_fr_FR.ts:2359]: Should be: "À propos de Qt"

**Review by jujudusud — CHANGES_REQUESTED** [review]

**Pull request merged**